doc/extension_store/manifest.jsonc

```diff
@@ -0,0 +1,139 @@
+{
+    // Info displayed in the Extension Store
+    "id": "postgres-vs",
+    "name": "PostgreSQL Virtual Schema",
+    "description": "...",
+    "version": "2.0.1",
+    // Dependencies
+    "bucketFsUpload": {
+        "jdbcDriver": {
+            // Download URL for the JDBC driver, e.g. Maven Central URL
+            "downloadUrl": "https://jdbc.postgresql.org/download/postgresql-42.3.3.jar",
+            // License URL
+            "licenseUrl": "https://github.com/pgjdbc/pgjdbc/blob/master/LICENSE",
+            // True if the user must agree to the license before installation, e.g. for Oracle JDBC driver
+            "licenseAgreementRequired": false,
+            // Filename when uploading files to BucketFS. Path is specific for the concrete extension version, e.g. `/bucketfs/<service>/<bucket>/extensions/<extension-id>/<extension-version>/`
+            "bucketFsFilename": "postgresql-jdbc-driver.jar"
+        },
+        "virtualSchemaAdapter": {
+            "downloadUrl": "https://github.com/exasol/postgresql-virtual-schema/releases/download/2.0.1/virtual-schema-dist-9.0.4-postgresql-2.0.1.jar",
+            "licenseUrl": "https://github.com/exasol/postgresql-virtual-schema/blob/main/LICENSE",
+            "bucketFsFilename": "postgresql-adapter.jar"
+        }
+    },
+    "parameters": {
+        "connectionName": {
```

Comment:
As a user I don't want to add a connection object but just fill in the credentials directly.
For me it's ok if the extension-manager picks a connection name automatically.